base9/Settings.cpp

```diff
@@ -521,11 +518,10 @@ template<typename T> T Settings::getOrRequest (Node & n, string && f)
     {
         string input = "";
         cout << "Please enter your desired setting for field: '" + f + "':\n";
-        getline(cin, input);
-        cout << "Field '" + f + "' set to: " << input << endl;
```

Comment:
(I suggest adding the logging feature to a new PR because my understanding is that it's best to separate PR's by features added, but if you think they should be lumped together that's fine too)